agent/rpm/pbench-agent.spec

```diff
@@ -0,0 +1,127 @@
+Name:           pbench-agent
+Version:     0.60
+%define gdist gc9e89f8f
+Release:     1%{?gdist}%{!?gdist:}
+Summary:        The pbench harness
+
+License:        GPLv3+
+URL:            http://perf1.perf.lab.eng.bos.redhat.com/atheurer/pbench
+Source0:        pbench-agent-%{version}.tar.gz
+Buildarch:      noarch
+
+
+%if 0%{?rhel} == 6
+%define turbostatpkg cpupowerutils
```

Comment:
These are tool RPM dependencies, we need to remove them from the agent RPM dependency list.